src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int MIN_SIZE = 1;
+    private static final int MAX_SIZE = 9;
+    private static final int DEFAULT_RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_RADIUS, DEFAULT_COLOR);
+    }
+
+    public Figure getRandomFigure(int randomIndex) {
+
+        switch (randomIndex) {
+            case 0:
+                return new Square(random.nextInt(MIN_SIZE, MAX_SIZE),
```

Comment:
Let's move this to a separate private method 
random.nextInt(MIN_SIZE, MAX_SIZE)
Also, let's use MAX_SIZE + 1, naming can be confusing since bound is excluded
![image](https://github.com/mate-academy/jv-oop-advanced/assets/49767049/057d1f33-6afc-42bd-ac42-27a19477f24a)